README.md

```diff
@@ -377,6 +377,29 @@ okta:
     rateLimit:
       maxRetries: 4
 ```
+
+When you use OAuth 2.0, the full YAML configuration would look like:
+
+```yaml
+okta:
+  client:
+    connectionTimeout: 30 # seconds
+    orgUrl: "https://{yourOktaDomain}" # i.e. https://dev-123456.oktapreview.com
+    proxy:
+      port: null
+      host: null
+      username: null
+      password: null
+    authorizationMode: "PrivateKey"
+    clientId: "{clientId}"
+    scopes:
+      - okta.clients.read
+      - okta.clients.manage
+    privateKey: "~/.okta/privateKey.pem" # PEM format is recommended. This SDK supports RSA AND EC algorithms - RS256, RS384, RS512, ES256, ES384, ES512.
```

Comment:
recommended probably isn't the right word here, as it is the only format supported currently.

Reply:
Also, we should probably have an example here that uses the PEM format inline too (maybe as the ENV var example) 🤔

Reply:
Checked with the team and it looks like we could choose which mode we'll support. Presently, I'd done changes to support a "path to pem file".

Reply:
👍 It's easy to add that in the future, we could probably also load a reference from the java keystone too (if people ask for it)